Mesh.session.sql

```diff
@@ -0,0 +1,4 @@
+DROp TAble app_accounts
```

Comment:
Why is this required?

Reply:
this wasa debug file i used, i will be dropping this